### PR TITLE
Update to the service type changes

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

FIxes: [645](https://github.com/ballerina-platform/ballerina-standard-library/issues/645)

### CODE REVIEW

File: task-ballerina/tests/task_test.bal [hunk 1]
```diff
@@ -28,8 +28,8 @@ function getCount() returns int {
     return count;
 }
 
-service appointmentService on appointment {
-    resource function onTrigger() {
+service /appointmentService on appointment {
+    resource function get onTrigger() {
```
Comment (daneshk) [inline task-ballerina/tests/task_test.bal:32]:
```suggestion
    remote function onTrigger() {
```

Comment (daneshk) [reply task-ballerina/tests/task_test.bal:32]: Shall we change it to `remote function` instead of `resource function`

Comment (kalaiyarasiganeshalingam) [reply task-ballerina/tests/task_test.bal]: Updated.

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**